diff -ru ../plone.app.iterate-1.2.3-orig/plone/app/iterate/browser/info.py ./plone/app/iterate/browser/info.py
--- ../plone.app.iterate-1.2.3-orig/plone/app/iterate/browser/info.py	2008-11-08 08:54:13.000000000 -0700
+++ ./plone/app/iterate/browser/info.py	2012-02-17 14:41:56.000000000 -0700
@@ -16,7 +16,7 @@
 
 from plone.app.iterate.util import get_storage
 from plone.app.iterate.interfaces import keys
-
+from plone.app.iterate.permissions import CheckoutPermission
 from plone.app.iterate.relation import WorkingCopyRelation
 
 from plone.memoize.instance import memoize
@@ -41,7 +41,7 @@
     def created( self ):
         time = self.properties.get( keys.checkout_time, DateTime() )
         util = getToolByName(self.context, 'translation_service')
-        return util.ulocalized_time(time, None, self.context, domain='plonelocales')
+        return util.ulocalized_time(time, True, self.context, domain='plonelocales')
 
     @memoize
     def creator( self ):
@@ -80,7 +80,7 @@
 
     def render(self):
         if self.working_copy() is not None and \
-            getSecurityManager().checkPermission(ModifyPortalContent, self.context):
+            getSecurityManager().checkPermission(CheckoutPermission, self.context):
             return self.template()
         else:
             return ""

